### DISCUSSION TITLE
v2.7.5.4

### DISCUSSION THREAD

**Discussion by Limraj** [SCADA-LTS/Scada-LTS#2647] [General]
## 

The changelist is listed below:

### 📄️ GitHub issues/pullrequest list overview:
1. #623 Fixed email validation in users details
2. #806 Unable to disable System event alarm User Login 
3. #2521 Text "double click to close" inside datapoint details without need 
4. #2544 Fixed 'not logged in' after logged
5. #2548 Fixed import graphics
6. #2550 The maximum alarm resets when it reaches 'return to normal'
7. #2551 Added functions to 'Event search' result table 
8. #2555 Fixed error save checkbox Disabled in Maintenance events
9. #2556 Corrected AddLangToUser migration 
10. #2557 Logging expcetion details while initializing MangoContextListener
11. #2559 Using the cache in the AlarmLevelController
12. #2560 Fixed cache UserComment type event
13. #2565 Fixed can't find an element with id/name
14. #2566 Fixed create new user
15. #2569 Fixed deleting non existent user profile
16. #2570 Added docker-compose.yml with tomcat debug mode
17. #2472 Fixed error read checkbox 'Roll' in Increment Data Point
18. #2573 Redirect home url after logged
19. #2576 Fixed deleting non existent users
20. #2577 Fixed deleting last watch list Error: TypeError, Cannot read properties of undefined 
21. #2581 Fixed rendering view component 'Image Chart'
22. #2583 Fixed save fullScreen in user for user non-admin
23. #2586 Added servlet httpds-secure
24. #2588 Fixed not emit sounds
25. #2591 Fixed 'Event Search' for 'Active' and 'Returned to normal' status
26. #2600 Use of relative links in Link View Component (#2597)
27. #2611 Fixed event message in Publisher
28. #2618 Improved stability for a large number of events
29. #2621 Fixed A server error has occured in Point details menu on alphanumeric data point
30. #2623 Fixed Gradle task 'deployTomcat' doesn't return error when build/libs folder is empty/deleted
31. #2625 Fixed validate Event Detectors
32. #2634 Fix possible Unreleased lock

<hr /><em>This discussion was created from the release <a href='https://github.com/SCADA-LTS/Scada-LTS/releases/tag/v2.7.5.4'>v2.7.5.4</a>.</em>

**Comment by fabiodurao** [comment]
in the datapoint details, the frames that showed which screens the variable is being used on and which users have which level of access to this variable disappeared, probably that disappeared when correcting the error below.

#2521 

![image](https://github.com/SCADA-LTS/Scada-LTS/assets/38673539/c1e1edb9-1c92-49b3-9a83-af3dbc1dd4b7)

**Comment by Limraj** [comment]
Hi @fabiodurao,
defect after task: #2544
I will fix in version 2.7.6.

I created issue:
https://github.com/SCADA-LTS/Scada-LTS/issues/2666

Thanks for your vigilance.

Regards,
Kamil Jarmusik
